gulpfile.js

```diff
@@ -0,0 +1,39 @@
+const gulp = require('gulp');
```

Comment:
For organization purposes I think it should better if this file and others about build process (android folder / ios folder) must be in the scripts folder.

See examples about this pattern:
https://github.com/facebook/react/tree/master/scripts
https://github.com/facebook/jest/tree/master/scripts